config/runtime.exs

```diff
@@ -3,7 +3,11 @@ import Config
 # Configure the type of names used for distribution and the node name.
 # By default a random short name is used.
 # config :livebook, :node, {:shortnames, "livebook"}
-# config :livebook, :node, {:longnames, "livebook@127.0.0.1"}
+# config :livebook, :node, {:longnames, :"livebook@127.0.0.1"}
+
+if System.get_env("CI") == "true" do
+  config :livebook, :node, {:longnames, :"livebook@127.0.0.1"}
+end
```

Comment:
Can we move this to config/test.exs instead?

Reply:
We can even have it in tests without the conditional, so it runs the same on our machines too.

Reply:
That's what I did initially, but then I thought it's actually CI specific config to solve CI specific problem. Also this way we test shortnames distribution locally and longnames distribution in the CI, so this does even improve coverage in a way ^^ Would you rather keep them the same?

Reply:
I would keep in config/test.exs with the conditional then to keep all test related concerns in a single place. :)

Reply:
Ah for a moment I thought we need it here to use the env variable, but in the CI w actually compile everything so this definitely makes sense!